exporter/containerimage/writer.go

```diff
@@ -432,6 +432,11 @@ func (ic *ImageWriter) commitDistributionManifest(ctx context.Context, opts *Ima
 		return nil, nil, err
 	}
 
+	remote, err = cache.PatchLayers(ctx, ref, opts.RefCfg, remote, sg)
+	if err != nil {
+		return nil, nil, err
+	}
+
```

Comment:
This exception should also be in `_nydus.go` file.

Reply:
Wrapped it to `exporter/containerimage/patch.go` -> `patchImageLayers`, PTAL.